src/lib/Scenes/Collection/__stories__/Collection.story.tsx

```diff
@@ -0,0 +1,26 @@
+import { storiesOf } from "@storybook/react-native"
+import React from "react"
+
+import { CollectionRenderer } from "lib/relay/QueryRenderers"
+import renderWithLoadProgress from "../../../utils/renderWithLoadProgress"
+import { CollectionContainer } from "../Collection"
+
+const CollectionStoryRenderer = ({ collectionID }) => {
+  return (
+    <CollectionRenderer
+      collectionID={collectionID}
+      render={renderWithLoadProgress(CollectionContainer, { collectionID })}
+    />
+  )
+}
+
+storiesOf("Collection")
```

Comment:
This simply renders a handful of different collections that are known to be used during development currently. 

This seems to be one use of Storybooks within this repo (see e.g. fairs and shows), in addition to the more typical use of developing components in isolation (see e.g. artworks).

👇 

![coll](https://user-images.githubusercontent.com/140521/72537356-058bba00-384a-11ea-8cbc-7d96fff36f18.gif)